.gitignore

```diff
@@ -33,3 +33,7 @@ alacritty_*_source.tar.bz2
 *.msi
```

Comment:
^ there is a legacy Flamegraph thing, should we delete it.

Reply:
You commented the wrong thing in GitHub, but yeah we probably should.

Reply:
I can't comment on a line that is not part of a diff, so I added `^`  thing.

Reply:
Oh, right. Gotta love GitHub.